keyboards/mechwild/waka60/readme.md

```diff
@@ -0,0 +1,19 @@
+# Waka60
+
+![waka](https://i.ibb.co/10cG7G2/image.png)
```

Comment:
Please rehost this on imgur, and add `h` to the end of the filename.

Reply:
Changed. When we uploaded that file there was some kind of drama regarding imgur and the permissions/longevity of the files that they were hosting.